### PR TITLE
most ability displays

### PR DESCRIPTION
**Pull request by stacktrace-error**
surgically removed the format markers from the existing names with unsurgical precision, should fix for most latinish alphabet languages but may break others

extremely stupid usage of `waves.shields` in shieldregenfield, I don't know whether that has to be added as its own bundle nor how to name it

image

image

image

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.
  - Tested with all vanilla units.

### CODE REVIEW

**Comment by stacktrace-error** [comment]
I don't have a fix for the minor text offset on empty displays rn, don't know whether there's something to use besides pad for it

**Comment by stacktrace-error** [comment]
should there be 2 abilities per line?

**Review by Darkness6030 — CHANGES_REQUESTED** [review]
What is this comment?

File: core/src/mindustry/entities/abilities/ShieldRegenFieldAbility.java [hunk 1]
```diff
@@ -1,9 +1,15 @@
 package mindustry.entities.abilities;
 
+import arc.Core;
+import arc.scene.ui.layout.Table;
 import arc.util.*;
 import mindustry.content.*;
 import mindustry.entities.*;
 import mindustry.gen.*;
+import mindustry.world.meta.Stat;
```
Comment (Anuken) [inline core/src/mindustry/entities/abilities/ShieldRegenFieldAbility.java:9]: use wildcard imports everywhere

**Pull request merged**